x/staking/keeper/genesis.go

```diff
@@ -26,9 +26,10 @@ func (k Keeper) InitGenesis(ctx context.Context, data *types.GenesisState) ([]ap
 	// initialized for the validator set e.g. with a one-block offset - the
 	// first TM block is at height 1, so state updates applied from
 	// genesis.json are in block 0.
-	sdkCtx := sdk.UnwrapSDKContext(ctx)
-	sdkCtx = sdkCtx.WithBlockHeight(1 - sdk.ValidatorUpdateDelay) // TODO: remove this need for WithBlockHeight
-	ctx = sdkCtx
+
+	// sdkCtx := sdk.UnwrapSDKContext(ctx)
```

Comment:
how can this get updated? I don't think we have a service for this.
cc @tac0turtle 

At least I don't think we should comment it, we may miss it for baseapp.

Reply:
should we just check if we can unwrap it? that way in v1 its there and in v2 its not?